hydromt_fiat/workflows/equity_data.py

```diff
@@ -213,4 +217,10 @@ def get_block_groups(self):
     
     def clean(self):
         """Removes unnecessary columns"""
-        self.equity_data_shp = self.equity_data_shp[["GEOID_short", "TotalPopulationBG", "PerCapitaIncomeBG"]]
\ No newline at end of file
+        self.equity_data_shp = self.equity_data_shp[["GEOID_short", "TotalPopulationBG", "PerCapitaIncomeBG"]]
+
+    def rename_geoid_short(self):
+        # Create string from GEO_ID short 
+        for index, row in self.equity_data_shp.iterrows():
+            BG_string = f"BG: {row['GEOID_short']}"
+            self.equity_data_shp.at[index, 'GEOID_short'] = BG_string
```

Comment:
this could be a one liner with:

`self.svi_data_shp['GEOID_short'] = "BG: " + self.svi_data_shp['GEOID_short']`

Reply:
the one-liner duplicated the str "BG" in one row for some reason so kept the code in its original form

"BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: BG: ....450190001001"